### PR TITLE
Serve autohinted fonts to blink on XP

### PR DESCRIPTION
**Pull request by sndrs**
n.b. this needs the font to be uploaded to pasteup-prod to work in prod, which I'm waiting for perms to do. 

---

On XP with font-smoothing disabled, blink renders our fonts like this: 
chrome xp on off

this adds a new auto-hinted version of the font which is only served to blink on XP and looks like this:
chrome xp auto off

Other versions of windows  have font-smoothing enabled by default, so it is not needed. XP has font-smoothing enabled after service pack 1, but not before, and we cannot know if has been installed. 

This _doesn't_ fix blink on later versions of windows with font-smoothing disabled - or for that matter ff on non-XP with font-smoothing disabled. In fact, auto-hinted fonts don't fix either of them in those conditions. 

But have to disable a default setting that will also degrade your whole experience to create that situation though, so it seems less of a problem...

### CODE REVIEW

**Comment by robertberry** [comment]
Wowsa that's a lot of files

**Comment by phamann** [comment]
tl;dr :+1:

**Comment by sndrs** [comment]
TOO MANY FILEZ
